website/docs/registry/policy-libraries/publishing.mdx

```diff
@@ -0,0 +1,85 @@
+
+### Sentinel File
+
+The GitHub repository must include a `sentinel.hcl` file at its root level that contains the names and file locations for the policies and policy modules included in the policy library. The Terraform Registry ignores any policy files in the repository that are not listed in the file.
+
+The `sentinel.hcl` file must follow the [Sentinel CLI Configuration File Syntax](https://docs.hashicorp.com/sentinel/configuration) with two deviations:
+
+-  [Remote sources](https://docs.hashicorp.com/sentinel/configuration/remote-sources) for policies and policy modules are not allowed.
+- The `enforcement_level` for a policy is not required. The Terraform Registry ignores this information if included.
+
+### Version Tags
+
+The Terraform Registry uses tags to identify versions of policy libraries. Version tag names must be a [semantic version](http://semver.org/), which can optionally be prefixed with a `v`. For example, `v1.0.4` and `0.9.2` are both acceptable. To publish a new policy library, at least one acceptable tag must be present. The Terraform Registry ignores tags that do not match these conventions.
+
+## Documenting a Policy Library
+
+The Terraform Registry displays documentation for policy libraries from Markdown files in your GitHub repository.
+
+### Directory Structure
+
+Markdown documents should be in the following locations within the repository. The Terraform Registry ignores Markdown files stored elsewhere.
+
+| Location | Filename | Description |
+| --- | --- | --- |
+| `/` | `README.md` | Index page for the policy library. |
+| `docs/policies/` | `<policy name>.md` | Description of a single policy. |
+| `docs/modules/` | `<module name>.md` | Description of a single policy module. |
+
+The `<policy name>` and `<module name>` in the Markdown filenames must match the names of `policy` and `module` blocks in the library's `sentinel.hcl` file.
+
+## Publishing to the Registry
+
+Before you publish, [review the policy library categories on the Terraform Registry](https://registry.terraform.io/browse/policies). Categories are general themes that can help users find policies that are relevant to their use case. We recommend designing your policy libraries to target a specific category, like networking rules or storage security. This helps the community find useful policies more easily.
+
+To publish a policy library to the Terraform Registry:
+
+1. Ensure that your policy library's repository meets the [requirements](#requirements).
+1. If a GitHub organization owns the repository, verify that the Terraform Registry OAuth app has access to the organization. Go to your [GitHub Settings](https://github.com/settings/applications) and select the Terraform Registry Application under **Authorized OAuth Apps** to confirm.
+1. [Sign in to the Terraform Registry](/registry#user-account) with a GitHub account.
+1. Select **[Publish > Policy Library](https://registry.terraform.io/publish/policy)**.
```

Comment:
I just wonder how helpful the link is --> this publish button is readily available on the homepage of the registry, right?

```suggestion
1. Select **Publish > Policy Library**.
```